SignOutSessionStateManager (#604)
bUnit-dev/bUnit: CHANGELOG.md

## [Unreleased]

> -   Added fake version of the `SignOutSessionStateManage` type in Blazor that makes it possible to test components that use the type. By [@egil](https://github.com/egil) and [@linkdotnet](https://github.com/linkdotnet).

Reviewer: ```suggestion
- Added `FakeSignOutSessionStateManage` type in Blazor, that makes it it easy to test components that use the `SignOutSessionStateManage` type. By [@linkdotnet](https://github.com/linkdotnet).
```

No need for to credit me for this. Its all you!

Author: > No need for to credit me for this. Its all you!
That is very generous but still you added very valuable feedback plus implementation details. I personally benefited a lot from your feedback and therefore think that you contributed a good chunk of the work.
